src/routes/expenses.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
Consider removing the 'use strict' directive as it is unnecessary in ECMAScript modules.
But don't change it here